wiki/Tournaments/Zotcade/2019/en.md

```diff
@@ -0,0 +1,294 @@
+---
+tags:
+- iDOLANSTER
+- Irvine
+- LAN
+- 'The iDOL@NSTER: Irvine Spring Stage'
+- UCI
+- Zotcade 2019
+---
+
+# iDOL@NSTER
+
+![iDOL@NSTER Logo](img/logo.png)
+
+The **iDOL@NSTER** is an osu! and osu!mania BYOC LAN tournament hosted by [osu! UCI](/wiki/Organisations/osu!_UCI) in coordination with [The Association of Gamers at UC Irvine](https://www.facebook.com/TAGatUCI/). It will take place during *Zotcade 2019*, a gaming festival hosted at the University of California, Irvine (*UCI*). This is osu! UCI's fourth installment of yearly LAN tournaments at Zotcade.
+
+## Event Details
+
+![Map of the Student Center showing Moss Cove](img/student-center-map.png "Moss Cove in the Student Center")
+
+The tournament will take place on Sunday, May 19, 2019 from 10 a.m. to 10 p.m. It will be located in the UCI Student Center (full address: 311 W Peltason Dr, Irvine, CA 92697), room Moss Cove AB. Food is not provided at the event, but there are plenty of chains and restaurants in walking distance around the area. All-day parking is available in the adjacent Student Center Parking Structure for around $10. Please arrive early, as there will be other events taking up parking spots.
+
+Each player location has one guaranteed power outlet and ethernet port. If you require the use of ethernet, please bring your own cable.
+
+### Registration
+
+Please fill out [this Google form](https://docs.google.com/forms/d/e/1FAIpQLSfhMFC1BZrbdpQK_tjpgYFjCGp4lUG8ILscpYH7XHvcLCmvdw/viewform) to RSVP and register for the event.
+
+## Tournament Schedule
+
+![iDOL@NSTER Schedule](img/schedule.png)
+
+## Prizes
+
+All prizes will be awarded to both the osu! and osu!mania winners.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile badge, HyperX Cloud Alpha Headset |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | HyperX Alloy Core RGB Keyboard |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | HyperX Pulsefire FPS Mouse |
+
+![iDOL@NSTER Profile Badge](img/badge.png)
+
+## Organisation
+
+The iDOL@NSTER is run by various osu! community members at UCI or visiting for the event.
```

Comment:
```suggestion
The iDOL@NSTER is run by various osu! community members at UCI or visiting the event.
```

Reply:
what's wrong with the for? you're changing the meaning of the sentence with this suggestion